platform/o.n.bootstrap/src/org/netbeans/JarClassLoader.java

```diff
@@ -272,6 +289,12 @@ public Map<String, Attributes> getEntries() {
                     }
                 }
                 Manifest man = new DelayedManifest();
+                
+                src.setMultiRelease(man.getMainAttributes().containsKey(MULTI_RELEASE));
```

Comment:
The value of the attribute needs to be checked, that it equals case-insensitive to `true`

Reply:
Fixed.